### ISSUE TITLE
Create an API for Ardere

### ISSUE THREAD

**Issue by jrconlin** [loads/ardere#57]
API should handle things like cluster management, test template construction, run management, and report collection.

This API would be used by a future CLI.

**Comment by rpappalax** [comment]
run management - need an easy way to kill a run

**Comment by jrconlin** [comment]
Absolutely, the mgmt layer would focus on (at least) three many functions:

configuration (building from templates and configs to allow for high reuse of work)
control (run, pause, adjustment, and early termination of jobs)
collection (gather run results and logs with some garbage filtering, data merge, etc.)

Looking at other elements, so as you can think of them, PLEASE ADD!
